<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CallWindow</class>
 <widget class="QMainWindow" name="CallWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>640</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>1</horstretch>
    <verstretch>1</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="2" column="7">
     <widget class="QPushButton" name="settings">
      <property name="toolTip">
       <string extracomment="Settings"/>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
    </item>
    <item row="2" column="8">
     <widget class="QPushButton" name="stats">
      <property name="text">
       <string>Statistics</string>
      </property>
     </widget>
    </item>
    <item row="2" column="5">
     <widget class="QPushButton" name="mic">
      <property name="toolTip">
       <string extracomment="Mic off"/>
      </property>
      <property name="text">
       <string>Mic off</string>
      </property>
     </widget>
    </item>
    <item row="2" column="6">
     <widget class="QPushButton" name="camera">
      <property name="text">
       <string>Camera off</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="addParticipant">
      <property name="text">
       <string>Add participant</string>
      </property>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="EndCallButton">
      <property name="toolTip">
       <string extracomment="End call"/>
      </property>
      <property name="text">
       <string>End call</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" rowspan="2" colspan="8">
     <widget class="QWidget" name="participants" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>320</width>
        <height>240</height>
       </size>
      </property>
      <layout class="QGridLayout" name="participantLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="spacing">
        <number>0</number>
       </property>
      </layout>
     </widget>
    </item>
    <item row="2" column="9">
     <widget class="VideoWidget" name="SelfView" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>320</width>
        <height>240</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QLineEdit" name="ip">
      <property name="maximumSize">
       <size>
        <width>124</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="text">
       <string>127.0.0.1</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAdd_Caller">
   <property name="text">
    <string>Add Caller</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QWidget</extends>
   <header>videowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>EndCallButton</sender>
   <signal>clicked()</signal>
   <receiver>CallWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>385</x>
     <y>561</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addParticipant</sender>
   <signal>clicked()</signal>
   <receiver>CallWindow</receiver>
   <slot>addParticipant()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>640</x>
     <y>581</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>mic</sender>
   <signal>clicked()</signal>
   <receiver>CallWindow</receiver>
   <slot>micState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>620</x>
     <y>585</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>camera</sender>
   <signal>clicked()</signal>
   <receiver>CallWindow</receiver>
   <slot>cameraState()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>756</x>
     <y>585</y>
    </hint>
    <hint type="destinationlabel">
     <x>639</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addParticipant()</slot>
  <slot>micState()</slot>
  <slot>cameraState()</slot>
 </slots>
</ui>
